apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -241,8 +241,13 @@ function load_blog_posts_block() {
  * Load WPCOM Block Editor NUX
  */
 function load_wpcom_block_editor_nux() {
-
-	require_once __DIR__ . '/wpcom-block-editor-nux/class-wpcom-block-editor-nux.php';
+	// TODO: if ( apply_filters( 'a8c_enable_wpcom_welcome_tour', false ) ) {
+	if ( true ) {
```

Comment:
Before we merge we can replace this `true` with something like the following:

```suggestion
	if ( defined( 'WPCOM_WELCOME_TOUR' ) && WPCOM_WELCOME_TOUR ||
		apply_filters( 'a8c_enable_wpcom_welcome_tour', false )
		) {
```
We did this [elsewhere](https://github.com/Automattic/wp-calypso/blob/trunk/apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php#L335).

The reason is that we can merged the code and, so long as we haven't turned on the `a8c_enable_wpcom_welcome_tour` filter in WPCOM, we can continue to hide the plugin behind the `WPCOM_WELCOME_TOUR` constant, which we can define on our sandboxes for testing.

Reply:
Yes!! Was already setting up the `WPCOM_WELCOME_TOUR` conditional. :)

I also wanted to set that constant on my ephemeral site, but am unable to ssh into it. So I just need to make a build with this `if (true)` hack for the ephemeral site, then I'll remove the hacks for merging.